scripts/preprocess_spec.py

```diff
@@ -16,24 +16,63 @@ def _to_camel_case(s):
     return ''.join(_title(y) for y in s.split("_"))
 
 
-def iterate_through_operations(spec, func):
+def iterate_through_operations(spec, *funcs):
```

Comment:
(No action required) 

'iterate' usually implies an iterator, which this is not.  Consider renaming to something like 'apply_funcs_to_spec'.